"""try:
    print(deneme)
except NameError:
    print("isim hatası oluştu")
except:
    print("hata oluştu")
else:
    print("hatasız çalıştı")
finally:
    print("final")
    """

x = -1

"""if x < 0:
    raise Exception("0'ın altında bir rakam yok")
    raise TypeError("0'ın altında bir rakam yok")"""

"""y = {"1":"aa"} """  """KeyError"""

""""""
"""print(y["2"])
TypeError
print("1" + 1)"""